src/main/resources/assets/buildinggadgets/lang/en_us.lang

```diff
@@ -126,4 +128,4 @@ config.buildinggadgets.general.subCategoryGadgets.gadgetDestruction.maxEnergy=Ma
 config.buildinggadgets.general.subCategoryGadgets.gadgetDestruction.maxEnergy.tooltip=The max energy of the Destruction Gadget
 config.buildinggadgets.general.subCategoryGadgets.gadgetCopyPaste=Copy-Paste Gadget
 config.buildinggadgets.general.subCategoryGadgets.gadgetCopyPaste.tooltip=Energy Cost & Durability of the Copy-Paste Gadget
-
```

Comment:
The names and comments for the newly added paste container configs need to be localized.